packages/manager/modules/pci/src/projects/project/public-gateways/translations/Messages_fr_FR.json

```
@@ -1,3 +1,14 @@
 {
-  "pci_projects_project_public_gateway_title": "Public Gateways"
+  "pci_projects_project_public_gateway_title": "Public Gateways",
+  "pci_projects_project_public_gateways_intro": "Les Public Gateway fournissent une connectivité aux instances privées, permettant d’accéder de manière sécurisée aux réseaux externes ou d’exposer l'application à un accès externe. Dans le premier cas, Public Gateway agit comme une passerelle SNAT, autorisant les instances à atteindre Internet, mais pas le contraire. Dans le second exemple, il s’agit de donner la possibilité d’exposer un service derrière une instance privée ou un Load Balancer à Internet en utilisant des IP flottantes.",
```

May be we can split this phrase into small phrases to keep the idea behind the phrase, which is enumeration.
Probably `ul li` display is better than a paragraphe:
```suggestion
  "pci_projects_project_public_gateways_intro_part_1": "Les Public Gateway fournissent une connectivité aux instances privées, permettant d’accéder de manière sécurisée aux réseaux externes ou d’exposer l'application à un accès externe :",
  "pci_projects_project_public_gateways_intro_part_2":  "Dans le premier cas, Public Gateway agit comme une passerelle SNAT, autorisant les instances à atteindre Internet, mais pas le contraire.",
  "pci_projects_project_public_gateways_intro_part_3": "Dans le second exemple, il s’agit de donner la possibilité d’exposer un service derrière une instance privée ou un Load Balancer à Internet en utilisant des IP flottantes.",
```
To confirm with PU

---

confirmed from PU and updated.